md-dao/src/main/java/com/wipro/ats/bdre/md/dao/InstanceExecDAO.java

```diff
@@ -180,8 +181,15 @@ public void delete(Long id) {
                 int total=instanceExecList.size();
                 for (InstanceExec instanceExec:instanceExecList)
                 {
+                    if(instanceExec.getEndTs()!=null){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)